atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1841,6 +1856,15 @@ protected KeyValueService getKeyValueService() {
         return tableRefToCells;
     }
 
+    private Timer getTimer(String name) {
+        return metricRegistry.getTimers()
+                .getOrDefault(name, metricRegistry.timer(MetricRegistry.name(SnapshotTransaction.class, name)));
```

Comment:
why are we using `getOrDefault` rather than just `return metricRegistry.timer(MetricRegistry.name(SnapshotTransaction.class, name);`